### PR TITLE
add Tasks, fixes #839

### LINKED ISSUE #839: Add Tasks
[https://github.com/m1guelpiedrafita/Tasks](https://github.com/m1guelpiedrafita/Tasks).
[Demo](https://demo.miguelpiedrafita.com/tasks).
`PHP`

### PR DESCRIPTION
**Pull request by hebbet**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1002,6 +1002,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Restyaboard](http://restya.com/board/) - Open source Trello-like kanban board. ([Demo](http://restya.com/board/demo.html), [Source Code](https://github.com/RestyaPlatform/board)) `OSLv3` `PHP`
   * [TaskBoard](http://taskboard.matthewross.me/) - A Kanban-inspired app for keeping track of things that need to get done. ([Source Code](https://github.com/kiswa/TaskBoard)) `MIT` `PHP`
   * [Taskfreak](http://www.taskfreak.com/original) - A simple but efficient web based task manager written in PHP. ([Demo](http://demo.taskfreak.com/)) `GPLv3` `PHP`
+  * [Tasks](https://github.com/m1guelpiedrafita/Tasks) - Simple tasks & notes manager written in PHP, jQuery and Bootstrap using a custom flat file database. ([Demo](http://projects.miguelpiedrafita.com/Tasks/) `MPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: Please change the homepage URL to `https://projects.miguelpiedrafita.com/Tasks/`, change the Demo URL to `https://demo.miguelpiedrafita.com/tasks/` and add the GitHub URL in a Source Code section.

**Comment by hebbet** [comment]
done.

File: README.md [hunk 2]
```diff
@@ -1002,6 +1002,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Restyaboard](http://restya.com/board/) - Open source Trello-like kanban board. ([Demo](http://restya.com/board/demo.html), [Source Code](https://github.com/RestyaPlatform/board)) `OSLv3` `PHP`
   * [TaskBoard](http://taskboard.matthewross.me/) - A Kanban-inspired app for keeping track of things that need to get done. ([Source Code](https://github.com/kiswa/TaskBoard)) `MIT` `PHP`
   * [Taskfreak](http://www.taskfreak.com/original) - A simple but efficient web based task manager written in PHP. ([Demo](http://demo.taskfreak.com/)) `GPLv3` `PHP`
+  * [Tasks](https://projects.miguelpiedrafita.com/Tasks/) - Simple tasks & notes manager written in PHP, jQuery and Bootstrap using a custom flat file database. ([Demo](https://demo.miguelpiedrafita.com/tasks/), ([Source](https://github.com/m1guelpiedrafita/tasks)) `MPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: The 'Source' label needs to be changed to 'Source Code'

**Comment by hebbet** [comment]
done

**Pull request merged**